### ISSUE TITLE
notebooks in cloned repos are read-only

### ISSUE THREAD

**Issue by rokroskar** [SwissDataScienceCenter/renga#170]
@ciyer reports that when a notebook is launched and the repo is automatically cloned in the attached volume, the permissions on the directory are not set correctly.

**Comment by rokroskar** [comment]
seems to be related to this error in the jupyterhub logs:

```
jupyterhub_1     | [I 2018-04-13 23:23:37.874 JupyterHub spawners:195] b'fatal: reference is not a tree: a6d3f4b3df6d822ad7656c1d6d48c603e4450e04\n'
```

The cloned git repo only contains the `.git` directory initially.

**Comment by rokroskar** [comment]
closed via #173
